script/libtest.bash

```diff
@@ -161,8 +161,10 @@ _wait_for_kubeless_kafka_server_ready() {
     echo_info "Waiting for kafka-0 to be ready ..."
     k8s_wait_for_pod_logline "Kafka.*Server.*started" -n kubeless kafka-0
     sleep 10
-    kubeless topic create "${test_topic}" || true
-    _wait_for_kubeless_kafka_topic_ready "${test_topic}"
+    kubeless topic list | grep -qw "${test_topic}" || {
```

Comment:
wouldn't be better to set a random topic to `test_topic`?

Reply:
Agreed.

Reply:
now you don't need to change these two lines since the topic is random :)